core/src/main/java/com/amplifyframework/predictions/models/TextFeature.java

```diff
@@ -28,20 +28,41 @@
  */
 @SuppressWarnings("unchecked")
 public abstract class TextFeature<T> extends Feature<T> {
```

Comment:
We'd talked about calling this like `TestPortion` or `TextSelection` or `TextSection` or `TextSegment` or something ... now that it has the positioning info embedded, worth revisiting? Or no?

Reply:
Hmm, are you talking about renaming the `TextFeature` class? It is still a feature, so I would like to keep that word in the name. I believe our old discussion was regarding the name of `TargetText` class, but that on longer exists.